src/workingtitle-vcockpits-instruments/html_ui/Pages/VCockpit/Instruments/Shared/Map/WorkingTitle/Svg/SvgRangeCompass.js

```diff
@@ -0,0 +1,27 @@
+class SvgRangeCompass extends SvgMapElement {
```

Comment:
Is this the start of an attempt to draw range rings on the map?

Reply:
Yes. I forgot that file was in there when I did the mass copy-paste.

Reply:
Haha, no worries.   It's something that's been requested  of us already, so I don't mind having the stub there.  Hopefully we'll be able to flesh it out sooner rather than later.